packages/common-scripts/tests/common.test.ts

```diff
@@ -32,6 +34,29 @@ test.before(() => {
     throw new Error("can not find bigint");
   };
 });
+let callCount = 0;
+sinon.stub(CellCollector.prototype, "collect").callsFake(async function* () {
+  callCount++;
+  if (callCount === 1) {
```

Comment:
i think this PR is just a lint fix, why tests must change here?

Reply:
use `sinon stub` and `callCount `is wrong here,I change bank to use `MockCellCollector`